data/map planets.txt

```diff
@@ -2394,6 +2396,25 @@ planet "Kuwaru Efreti"
 	bribe 0
 	security 1
 
+planet "La'ab Bamen"
+	attributes korath "korath exile" "slavery ban" "Independent (Korath-Allied)"
+	landscape land/monterey-bay-aquarium
+	# This station was constructed by the Korath, hence the large size and advanced technology.
+	# The fish within are from the New Tortuga Korath fish trade. You're looking at the
+	# juvenile and adult forms, swimming in the extremely specific environment they require
+	# to survive.
+	description `This moon is only a few kilometers in diameter, and it has been hollowed out and turned into an aquarium. There's several cubic kilometers of salt water inside, with numerous creatures that look like small translucent glowing dolphins, and a few that look like a hybrid of an ankylorhiza and a giant squid. A machine at the moon's center maintains precise conditions, including typical ocean temperatures and a slow electric current. There are corridors throughout this asteroid aquarium, lit by a glow from the water itself.`
+	description `At the surface of the asteroid, there are hidden hangar doors disguised as craters, leading to bays designed for ships larger than anything the human race has ever produced. The facility looks like it was constructed from human technology, but at an impossibly large scale.`
+	spaceport `This spaceport is tightly compartmentalized, with zones carefully marked out for certain purposes. Each zone can only be accessed by an elevator-like system that moves both horizontally and vertically. With this setup, it is nearly impossible for one group of visitors to see another inside the spaceport. The only shared area is the gigantic hangar bays, which are few in number, but those too seem to be divided by purpose.`
+	government "Independent (Korath-Allied) Restricted"
+	"required reputation" 100
+	bribe 0.04
+	security 0
+	tribute 1200
+		threshold 7000
+		fleet "Independent (Korath-Allied) Defense" 7
+		fleet "Korath Humanika Tribute Response" 3
```

Comment:
```suggestion
		fleet "Korath Tribute Response (Humanika Efreti)" 3
```

Reply:
Corrected to "Humanika Rafeki" and manually applied.